go.mod

```diff
@@ -153,11 +153,14 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 )
 
-replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
-
-replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-
-replace github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
-
-// TODO: remove once v0.45 has been released
-replace github.com/cosmos/cosmos-sdk => github.com/tharsis/cosmos-sdk v0.44.3-olympus
+replace (
+	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
+	github.com/confio/ics23/go => github.com/confio/ics23/go v0.6.6
+	github.com/cosmos/cosmos-sdk => github.com/tharsis/cosmos-sdk v0.44.3-olympus
+	github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.3
```

Comment:
@fedekunze note if you use dependant this will override that work. Multiple versions in the go.sum is safe 

https://go.dev/ref/mod#minimal-version-selection

Reply:
Yeah, so, this may have been misguided.  

I'll kinda do that whole command thing here, and post the result.  Was an interesting learning adventure, anyhow.

Reply:
cosmos/gaia#1099 (comment)

So basically, I'll do the same thing here, and then make a pr that un-pr's my pr.

Sorry about the confusion on that.